packages/block-library/src/block/edit.js

```diff
@@ -45,12 +93,20 @@ export default function ReusableBlockEdit( { attributes: { ref } } ) {
 		ref
 	);
 
+	const { alignment, layout } = useInferredLayout( blocks, parentLayout );
+	const layoutClasses = useLayoutClasses( { layout }, name );
```

Comment:
Generating the layout styles directly allows us to avoid;
- fully adopting layout support for the editor only
- setting layout attributes and needing to mark the changes as not persisted etc.